test/monitoring/monitoring.go

```diff
@@ -86,12 +83,10 @@ func PortForward(logf logging.FormatLogger, podList *v1.PodList, localPort, remo
 	return portFwdProcess.Pid, nil
 }
 
-// RunBackground starts a background process and returns the Process if succeed
-func executeCmdBackground(logf logging.FormatLogger, format string, args ...interface{}) (*os.Process, error) {
-	cmd := fmt.Sprintf(format, args...)
+// executeCmdBackground starts a background process and returns the Process if succeed
+func executeCmdBackground(logf logging.FormatLogger, cmd string, args ...string) (*os.Process, error) {
```

Comment:
If this is really only used in that one place, maybe we should just drop the function at this point? The wrapper is **really** thin.

Reply:
yeah, let me see the downstream tests pass with this and I'll tidy up a bit more

Reply:
hm actually I was hoping the downstream would run the eventing e2es with this for me, but looks like they don't anyway, bah